### ISSUE TITLE
Cant disable touch controls on android.

### ISSUE THREAD

**Issue by paravane** [Admiral-Billy/Pokerogue-App#23]

**Comment by Admiral-Billy** [comment]
I think Sam disabled the ability for mobile devices to disable touch controls to prevent softlocks; this may or may not get changed in the future.

**Comment by Soma257** [comment]
Solution: 
1. open pokerogue in chrome on mobile
2. tap on the 3 dots to open the menu
3. switch to desktop view 
4. after you got desktop view tap the 3 dots again
5. click on install app
6. now you can open pokerogue directly from you home screen and it wont have the chrome bezels
7. log in and open the menu, you will now be able to disable the mobile touch controls
Make sure your controller works before since this can softlock you if it dosen't and do not disable controller as that will also cause a softlock. If you got softlocked you would have to uninstall the app, clear the data and just redo it from step 1

while not a fix its atleast a way around it

**Comment by maximehuran** [comment]
```
click on install app
```

On Ayn Odin 2 for example we don't have this option.
This is why we use the app.
It could be nice to be able to disable touch control, then display a window with this message : 

```
Touch control are disable, if you don't confirm this operation, they will be back in 10s...9s...8s...

[Cancel] [Confirm]
```

This window will ensure the user that the D-pad controller work and the confirm button works too.
It will avoid soft locks

**Comment by Admiral-Billy** [comment]
Yeah this is the suggestion I gave myself, although it hasn't been implemented by anyone yet.

**Comment by maximehuran** [comment]
I understand ! If I have some time I will create a pull request with this feature.

Thank you for your work

**Comment by maximehuran** [comment]
I searched how to change the game menu settings, in the JS files I only saw the menus for the desktop app.
How can you change the in-game menu like `Daily Run (Beta)` ?

**Comment by Admiral-Billy** [comment]
Yeah, my app itself doesn't modify the game's source code at all (besides bypassing login), so any changes along these lines would have to be on the main game repo.

**Comment by paravane** [comment]
I wonder if I just find a chromium style app that's designed for kiosks if that'd be a decent work around as well. I was hoping to do Offline runs honestly.

`Touch control are disable, if you don't confirm this operation, they will be back in 10s...9s...8s...

[Cancel] [Confirm]`


This seems like the way to do it honestly.
